### PR TITLE
Updated contrib with section on testing against a Spark cluster

### PR DESCRIPTION
**Pull request by rjrudin**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -150,6 +149,35 @@ You now have a Spark dataframe - try some commands out on it:
...
 
+# Testing against a local Spark cluster
+
+When you run PySpark, it will create its own Spark cluster. If you'd like to try against a separate Spark cluster
+that still runs on your local machine, perform the following steps:
+
+1. Use [sdkman to install Spark](https://sdkman.io/sdks#spark). Run `sdk install spark 3.4.1` since we are currently
+building against Spark 3.4.1.
+2. `cd ~/.sdkman/candidates/spark/current/sbin`, which is where sdkman will install Spark.
+3. Run `start-master.sh` to start a master Spark node.
```
Comment (BillFarber) [inline CONTRIBUTING.md:160]: this should be `./start-master.sh`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -150,6 +149,35 @@ You now have a Spark dataframe - try some commands out on it:
...
+4. `cd ../logs` and open the master log file that was created to find the address for the master node. It will be in a
+log message similar to `Starting Spark master at spark://NYWHYC3G0W:7077` - copy that address at the end of the message.
+5. `cd ../sbin`.
+6. Run `start-worker.sh spark://NYWHYC3G0W:7077`, changing that address as necessary.
```
Comment (BillFarber) [inline CONTRIBUTING.md:164]: same here

File: CONTRIBUTING.md (hunk 1 above)
Comment (rjrudin) [reply CONTRIBUTING.md:160]: I got away with "start-master.sh" as it was already on my PATH.

**Review by BillFarber — APPROVED** [review]
just a couple of small comments

**Review by BillFarber — APPROVED** [review]

**Pull request merged**